contracts/vc-vault-factory/src/contract.rs

```
@@ -0,0 +1,77 @@
+    /// Return true if `vault_address` was deployed by this factory.
+    fn is_vault(e: Env, vault_address: Address) -> bool;
+}
+
+#[contractimpl]
+impl VaultFactoryContract {
+    pub fn __constructor(e: Env, vault_init_meta: VaultInitMeta) {
+        storage::set_vault_init_meta(&e, &vault_init_meta);
+    }
+}
+
+fn derive_salt(e: &Env, user_salt: BytesN<32>, owner: &Address) -> BytesN<32> {
```

Clippy (1.96.0)

[warning] function `derive_salt` is never used

(warning)